src/Microsoft.TestPlatform.VsTestConsole.TranslationLayer/ConsoleParameters.cs

```diff
@@ -35,6 +35,11 @@ public ConsoleParameters(IFileHelper fileHelper)
             this.fileHelper = fileHelper;
         }
 
+        /// <summary>
+        /// Trace level for logs.
+        /// </summary>
+        public PlatformTraceLevel TraceLevel { get; set; } = PlatformTraceLevel.Verbose;
```

Comment:
Should I use System.Diagnostics.TraceLevel here OR Microsoft.VisualStudio.TestPlatform.ObjectModel.PlatformTraceLevel.

Internally we use PlatformTraceLevel wherever TraceLevel is not present. Example: NETSTANDARD1_4

Reply:
Removed dependency on platform abstractions.
Cant remove System.Diagnostics.TraceLevel as used by exe config trace level.